package.json

```diff
@@ -49,8 +50,18 @@
     "tape": "~4.0.0"
   },
   "scripts": {
-    "test": "tape test/*-test.js | faucet"
+    "install": "node-pre-gyp install --fallback-to-build",
+    "test": "tape test/*-test.js | faucet",
+    "prebuild": "node-pre-gyp rebuild && node-pre-gyp package"
   },
   "license": "MIT",
-  "gypfile": true
+  "gypfile": true,
+  "bundledDependencies": [
```

Comment:
because https://github.com/mapbox/node-pre-gyp#1-add-new-entries-to-your-packagejson, not sure why it's not just a dependency